KeenClient/KeenClient.h

```diff
@@ -203,54 +203,56 @@ typedef NSDictionary* (^KeenGlobalPropertiesBlock)(NSString *eventCollection);
 /**
  Call this to indiscriminately delete all events queued for sending.
  */
-+ (void)clearAllEvents;
++ (void)clearAllEvents DEPRECATED_MSG_ATTRIBUTE("Class method clearAllEvents is deprecated. Use instance method instead.");
```

Comment:
So these deprecations are the major (or only?) potentially breaking public interface changes, right? The additional KeenClient init functions for passing in implementations of the other modules (network/uploader/store) are all strictly additive, right? So I imagine the plan is to remove these class methods in the next major release, right?